INSTALL.md

```diff
@@ -35,12 +35,17 @@ Note: the Intel(R) oneAPI components listed here can be installed together throu
 
 https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit.html
 
+These dependencies can also be installed through the `conda` software, but doing so will require a few additional setup steps - see rest of the instructions for details.
```

Comment:
I think it is possible to add the link to a respective chapter and remove "see rest ...".

Reply:
Added a link.